### PR TITLE
Add more sections

### PR DESCRIPTION
**Pull request by martinkarugaba**
I have added all the works and the about-myself section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -127,5 +127,464 @@ <h2 class="uppercase text-4xl text-h2 font-medium">
...
+
+    <!--* PROJECTS CONTAINER -->
+    <section class="px-5 py-32 grid grid-cols-1 gap-y-24 bg-gray-50">
+      <!--* PROJECTS CONTAINER -->
+      <!--* project one -->
+      <div
+        class="w-[100%] p-4 border border-light-grey-2 bg-white rounded-2xl flex flex-col justify-center items-center"
+      >
+        <!--* project screen shot -->
+        <div class="w-full h-[50%] bg-white">
```
Comment (algerina) [inline index.html:139]: - [ ] For every card, kindly change the container from a `div` to a `section`.

File: index.html [hunk 2]
```diff
@@ -127,5 +127,464 @@ <h2 class="uppercase text-4xl text-h2 font-medium">
       </div>
     </section>
```
Comment (algerina) [inline index.html:128]: - [ ] The background image is not displayed, to match the Figma design you must use the same background image.

Your design
sat1

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @martinkarugaba , 

You've done an amazing job fir 👏 💯 👍
There are some issues that you still need to work on to go to the next project but you are almost there! 

## Highlights
  -  No Linter errors 
  -  Descriptive commits
  - Your project looks awesome!



## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


You can also consider:
  - [ ] Changing the name in the header text from `I am Mariane` to your name.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: index.html (hunk 2 above)
Comment (martinkarugaba) [reply index.html:128]: The background image seems to be working, I have made another commit though.

**Review by Debas-31 — APPROVED** [review]
> ## Hi @martinkarugaba   :wave:, 

# Status: Approved :heavy_check_mark:  :heavy_check_mark:  :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Figma template usage :heavy_check_mark:
- README file :heavy_check_mark: 
- UI design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by martinkarugaba** [comment]
Thank you @Debas-31

**Pull request merged**